Error when using `vagrant up dokku-windows`

### Description of problem

Starting from a fresh instance, with current HEAD at 58325234463451d17ce2dd1f72a81d1d2e683537, running `vagrant up dokku-windows` gives the following log:

```
    dokku-windows: Checking for stray cats under /usr/local/man...
    dokku-windows: Checking for stray cats under /var/cache/man/oldlocal...
    dokku-windows: 1 man subdirectory contained newer manual pages.
    dokku-windows: 1 manual page was added.
    dokku-windows: 0 stray cats were added.
    dokku-windows: 5 old database entries were purged.
    dokku-windows: make[1]: Leaving directory '/root/dokku'
    dokku-windows: sudo -E dokku plugin:install-dependencies --core
    dokku-windows: sudo -E dokku plugin:install --core
    dokku-windows:  !     You haven't deployed any applications yet
    dokku-windows:  !     Could not find nginx binary in $PATH or at '/usr/sbin/nginx'.
    dokku-windows: make: *** [Makefile:132: plugins] Error 1
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

I'm not too sure about the priority this should get. I cannot get tests to run at all on Windows, both vagrant versions fails (windows and "global"), and vscode too, and I'm not sure which one to try to debug :/ 

However this error seems related to recent changes so I'm opening it ;)

[log.txt](https://github.com/user-attachments/files/17116539/log.txt)



### Steps to reproduce

`vagrant up dokku-windows`

### dokku report $APP_NAME

-

### Additional information

_No response_

### Output of failing commands after running: dokku trace:off

_No response_

### Output of failing commands after running: dokku trace:on

_No response_

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -72,7 +72,7 @@ Vagrant::configure("2") do |config|
   config.vm.define "dokku-windows", autostart: false do |vm|
     vm.vm.hostname = "#{DOKKU_DOMAIN}"
     vm.vm.network :private_network, ip: DOKKU_IP
-    vm.vm.provision :shell, :inline => "export DEBIAN_FRONTEND=noninteractive && apt-get update -qq >/dev/null && apt-get -qq -y --no-install-recommends install git dos2unix >/dev/null"
+    vm.vm.provision :shell, :inline => "export DEBIAN_FRONTEND=noninteractive && apt-get update -qq >/dev/null && apt-get -qq -y --no-install-recommends install git dos2unix nginx >/dev/null"
     vm.vm.provision :shell, :inline => "cd /vagrant/ && export DOKKU_BRANCH=`git symbolic-ref -q --short HEAD 2>/dev/null` && export DOKKU_TAG=`git describe --tags --exact-match 2>/dev/null` && cd /root/ && cp /vagrant/bootstrap.sh ./ && dos2unix bootstrap.sh && bash bootstrap.sh"
   end
 

